static/js/people.js

```diff
@@ -702,15 +702,13 @@ function people_cmp(person1, person2) {
 }
 
 exports.get_rest_of_realm = function get_rest_of_realm() {
-    var people_minus_you = [];
+    var people_including_you = [];
     active_user_dict.each(function (person) {
-        if (!exports.is_current_user(person.email)) {
-            people_minus_you.push({email: person.email,
-                                   user_id: person.user_id,
-                                   full_name: person.full_name});
-        }
+        people_including_you.push({email: person.email,
+                               user_id: person.user_id,
+                               full_name: person.full_name});
     });
-    return people_minus_you.sort(people_cmp);
+    return people_including_you.sort(people_cmp);
 };
```

Comment:
I think this isn't the right way to make this change; note how it makes the name totally inaccurate :).  Any other caller using this would be broken.

Also, we probably want the current user to be tagged specially (with an automatic checkmark to start), since we will add them by default.